### DISCUSSION TITLE
Using  EZABL

### DISCUSSION THREAD

**Discussion by Watchur** [Jyers/Marlin#872] [General Discussion]
Hello. I am very new so forgive me if this has been addressed. I didn't see anything in a search. Does the current firmware support  EZABL? Their website states that they don't support the V2 because the LCD screen doesn't support babystepping but this appears to be available in the Jyers firmware. Has anyone used  EZABL successfully?

**Comment by JoeInDowney** [comment]
I also have an EZABL, Can I just add the config into the no probe version?

**Reply by Jyers** [reply]
You can certainly try, if you find anything not working, feel free to report back and I can see what I can do to better support that probe.

**Reply by JoeInDowney** [reply]
I tried to load the bin file in VS code and could not open it. When it did it was garbed.

**Reply by Cal-LF** [reply]
@JoeInDowney .bin file is the result of compiling. VS Code uses the downloaded github zip. 
[Compiling Steps.txt](https://github.com/Jyers/Marlin/files/6527317/Compiling.Steps.txt)

**Comment by stephenfeather** [comment]
>  they don't support the V2 because the LCD screen doesn't support babystepping

I cynically and now not so secretly think it's to get you to buy their screen replacement. All joking aside, their support guys have been great despite my snarkiness in submitting tickets.  My Ender 3v2 head mount has the screws in the wrong place so the default EZABL mount had the adjustment screw behind the nut. They had a new STL to me within a few hours and it worked great.

That being said...I have the EZABL pro and much prefer it to the mechanical probes. I had a discussion with their support and they are only designed to connect to the Z endstop.  Later today I'm going to open a feature request for ABS using the zstop sensors.  Will need some configuration.h and configuration_adv.h changes and I think it might be worth its own screen and binary release eventually.

Hoping to test it in this firmware later this weekend.  Don't want to put too much time in until its on the swappable head hardware.

**Reply by PrintedVisionz3D** [reply]
were any of you successful in compiling for EZABL?

**Reply by ahuczekdesign** [reply]
> Hoping to test it in this firmware later this weekend. Don't want to put too much time in until its on the swappable head hardware.

Hey any luck here? This was a while back so I'd be curious what your results were. I'd love to give Jyers a try as well as I'm using stock TH3d Firmware but looks like all of the releases thus far are for BL Touch only.
